### PR TITLE
Fix last checkpoint

### PR DESCRIPTION
**Pull request by SammyRamone**
## Description
Fixes https://github.com/DLR-RM/rl-baselines3-zoo/issues/141

Since there is no directly build in natural sort method I added one. Alternatively, the natsort (https://github.com/SethMMorton/natsort) library could be used, but I think this solution is better as it does not add another dependency and this sorting is only needed at this one place.

I did not add it to the changelog, since there are only the last versions listed. Should I add a "Release 1.2.0a0 (WIP)" section?

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [ ] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [ ] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [ ] I have ensured `make pytest` and `make type` both pass. (**required**)

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: enjoy.py [hunk 1]
```diff
@@ -101,6 +102,12 @@ def main():  # noqa: C901
         checkpoints = glob.glob(os.path.join(log_path, "rl_model_*_steps.zip"))
         if len(checkpoints) == 0:
             raise ValueError(f"No checkpoint found for {algo} on {env_id}, path: {log_path}")
+
+        def natsort(s):
+            # we need to sort the list in natural order, but there is no standard python function for it
+            return [int(t) if t.isdigit() else t.lower() for t in re.split(r"(\d+)", s)]
```
Comment (araffin) [inline enjoy.py:108]: why not something more specific to this use case?
I'm thinking of `int(checkpoint_path.split("_")[2]) # path follow the pattern "rl_model_*_steps.zip"`

**Comment by araffin** [comment]
Hello,
thanks for the PR =)

>I did not add it to the changelog, since there are only the last versions listed. Should I add a "Release 1.2.0a0 (WIP)" section?

yes, please do ;)

File: enjoy.py (hunk 1 above)
Comment (SammyRamone) [reply enjoy.py:108]: Yeah I just thought about sorting it in natural order, not that there is a simpler solution in this special case. Your proposed solution would also work, as long as the naming scheme does not change. But this is highly unlikely. I can change this, if you want.

Comment (SammyRamone) [reply enjoy.py:108]: Your solution is also better to understand for any reader, so I guess it would be better.

**Review by araffin — APPROVED** [review]
LGTM, thanks =)

**Pull request merged**